src/app/pages/administration/url-checker/url-checker.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Container,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import { useSnackbar } from 'notistack';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+import GetAppIcon from '@material-ui/icons/GetApp';
+
+import { ChplTextField } from 'components/util';
+import { utilStyles } from 'themes';
+import usePostUrlChecker from 'api/url-checker';
+
+const useStyles = makeStyles({
+  ...utilStyles,
+  titlePadding: {
+    paddingTop: '16px',
+    paddingBottom: '16px',
+  },
+});
+
+const validationSchema = yup.object({
+  url: yup.string()
+    .required('Field is required')
+    .url('Improper format (http://www.example.com)'),
+});
+
+function ChplUrlChecker() {
+  const { enqueueSnackbar } = useSnackbar();
+  const postUrlChecker = usePostUrlChecker();
```

Comment:
On components with only one API call, I usually do the object spreading syntax:
```suggestion
  const { data, isLoading, isSuccess }= usePostUrlChecker();
```
and then down in the `useEffect` all of the `postUrlChecker.` text can be removed. I think it makes it a little more clear to read, though it's certainly not required (and way more complicated if there are more than one API call)